03_build_installer.sh

```diff
@@ -18,6 +18,8 @@ extract_oc "${OPENSHIFT_RELEASE_IMAGE}"
 
 mkdir -p $OCP_DIR
 
+save_release_info ${OPENSHIFT_RELEASE_IMAGE} ${OCP_DIR}
+
```

Comment:
Perhaps I should remove OCP_DIR here - I was aligning with the style used here but noticed after that OCP_DIR is actually already consumed directly in several functions defined in `ocp_install_env.sh`